### PR TITLE
HS-187: Add HeartbeatConsumer, persist Minion

### PR DESCRIPTION
**Pull request by cgorantla**
## JIRA Link
https://issues.opennms.org/browse/HS-187

## Checklist
* [x] JIRA has been updated to 'in review'.
* [ ] Unit tests have been added as part of the PR.
* [x] Manual tests have been completed, and notes added to JIRA.
* [ ] Details have been provided to docs team (if necessary)
* [ ] Screenshot(s) included (applicable to changes with a visual impact).

### CODE REVIEW

**Comment by cgorantla** [comment]
```
[horizon-stream-core] 13:27:40.791 INFO  [minion-monitor-1] ECHO RESPONSE: node-id=minion-01; node-location=minion-location; duration=8ms
[horizon-stream-core] 13:27:53.317 INFO  [sink-consumer-0] Received heartbeat for Minion with id: minion-01 at location: minion-location
[horizon-stream-core] 13:28:23.316 INFO  [sink-consumer-0] Received heartbeat for Minion with id: minion-01 at location: minion-location
[horizon-stream-core] 13:28:53.317 INFO  [sink-consumer-0] Received heartbeat for Minion with id: minion-01 at location: minion-location
```

File: platform/ipc/heartbeat/src/main/java/org/opennms/ipc/heartbeat/HeartbeatConsumer.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        messageConsumerManager.registerConsumer(this);
+    }
+
+    @Override
+    public SinkModule<MinionIdentityDTO, MinionIdentityDTO> getModule() {
+        return new HeartbeatModule();
+    }
+
+    @Override
+    public void handleMessage(MinionIdentityDTO minionHandle) {
```
Comment (yangli2014) [inline platform/ipc/heartbeat/src/main/java/org/opennms/ipc/heartbeat/HeartbeatConsumer.java:65]: It is better to add unit tests to cover this method and make sure nobody breaks the logic here in the feature.

**Comment by yangli2014** [comment]
Tested and works as expected. Just one minor comment above and it seems we didn't write enough unit tests.

**Comment by cgorantla** [comment]
Closing this in favor of #155

**Pull request closed without merging**